src/components/PetTabs.tsx

```diff
@@ -24,10 +16,14 @@ export const PetTabs = (props: PetTabsProps) => {
           <TabsTrigger value="appointments">Appointments</TabsTrigger>
         </TabsList>
         <TabsContent value="basicInfo">
-          <BasicInfo pet={props} />
+          <BasicInfo {...pet} />
+        </TabsContent>
+        <TabsContent value="vaccinations">
```

Comment:
would be awesome to add <Suspense> or something to display while fetching the data